cmd/digit.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/quesurifn/go-digit/digit"
+	"github.com/spf13/cobra"
+)
+
+var port string
+
+var digitRunCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run the go-digit gin server",
+	Long:  `Runs the go-digit gin server. It will default to running on 3030 if a port flag (--port or -p) is not specified`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Hello world!")
```

Comment:
Implementing cobra allows us to split out running the server like this. So now it's available at `go run main.go run -p <port>`.